webui/src/pages/repositories/wizard/spark_quickstart_wizard.jsx

```diff
@@ -0,0 +1,81 @@
+import Wizard from "./wizard";
+import React, {useState} from "react";
+import {useAPI} from "../../../lib/hooks/api";
+import {config} from "../../../lib/api";
+import {Loading} from "../../../lib/components/controls";
+import Container from "react-bootstrap/Container";
+import Row from "react-bootstrap/Row";
+import Col from "react-bootstrap/Col";
+import {Spinner} from "react-bootstrap";
+import Alert from "react-bootstrap/Alert";
+import RepositoryCreateForm from "../../../lib/components/repositoryCreateForm";
+
+const RepositoryCreationStep = ({repoCreationError, createRepo, onCancel, onComplete}) => {
+    const {response, error: err, loading} = useAPI(() => config.getStorageConfig());
+    const [repoName, setRepoName] = useState("");
```

Comment:
nit:
```suggestion
    const [repoName, setRepoName] = useState('');
```

Also this file mixes the 2 quotes.